src/bindings/overlay.js

```diff
@@ -10,6 +10,7 @@ const allowed = {
   ],
   attributes: {
     global: ['title', 'aria-label', 'aria-valuetext', 'aria-moz-hint'],
+    button: ['accesskey'],
```

Comment:
I need this for XUL. Overall we may want to have different whitelist for XUL and HTML. I really think that bindings/html and bindings/xul directories make sense